src/Symfony/Component/DependencyInjection/LazyProxy/PhpDumper/DumperInterface.php

```diff
@@ -38,7 +38,7 @@ public function isProxyCandidate(Definition $definition);
      *
      * @return string
      */
-    public function getProxyFactoryCode(Definition $definition, $id, $methodName);
+    public function getProxyFactoryCode(Definition $definition, $id, $methodName = null);
```

Comment:
spotted meanwhile: not allowing null breaks our forward compat policy (being able to upgrade a 3.4 code so that it works with 4.0)